common/djangoapps/student/models.py

```diff
@@ -123,7 +123,6 @@ class AnonymousUserId(models.Model):
     user = models.ForeignKey(User, db_index=True)
     anonymous_user_id = models.CharField(unique=True, max_length=32)
     course_id = CourseKeyField(db_index=True, max_length=255, blank=True)
-    unique_together = (user, course_id)
```

Comment:
Fun times. Apparently, these were never actually unique together, because this option is supposed to be put inside the `class Meta` definition (and wasn't). So, no migration needed, because the constraint never existed.